applications/opibuilder/opibuilder-plugins/org.csstudio.opibuilder.rcp/src/org/csstudio/opibuilder/util/ConsoleServiceSSHelperImpl.java

```diff
@@ -117,10 +117,14 @@ private String getTimeString(){
      */
     @Override
     public void writeError(final String message){
+        
         switch (PreferencesHelper.getConsolePopupLevel()) {
         case ALL:
             popConsoleView();
             break;
+        case NO_CONSOLE:
+            OPIBuilderPlugin.getLogger().log(Level.WARNING, message);
```

Comment:
I agree that this would be nice, but unfortunately it doesn't compile as `ERROR` isn't a valid errorlevel. `WARNING` felt like the most appropriate from this list: https://docs.oracle.com/javase/7/docs/api/java/util/logging/Level.html. We could use `SEVERE` but that doesn't exist in [log4j world](http://www.slf4j.org/api/org/apache/log4j/Level.html) so would get logged as `INFO` anyway - I preferred to stick with levels that exist in both...